tools/docker/builder/openwrt/build.sh

```diff
@@ -46,15 +48,20 @@ build_prplmesh() {
     trap 'docker rm -f $container_name' EXIT
     docker run -i \
            --name "$container_name" \
-           -e TARGET \
+           -e TARGET_SYSTEM \
+           -e SUBTARGET \
+           -e TARGET_PROFILE \
            -e OPENWRT_VERSION \
            -e PRPLMESH_VERSION \
-           -v "$scriptdir/scripts:/home/openwrt/openwrt_sdk/build_scripts/:ro" \
+           -v "$scriptdir/scripts:/home/openwrt/openwrt/build_scripts/:ro" \
            -v "${rootdir}:/home/openwrt/prplMesh_source:ro" \
            "$image_tag" \
            ./build_scripts/build.sh
     mkdir -p "$build_dir"
-    docker cp "${container_name}:/home/openwrt/openwrt_sdk/prplmesh-${TARGET}-${OPENWRT_VERSION}-${PRPLMESH_VERSION}.ipk" "$build_dir"
+    # Note: docker cp does not support globbing, so we need to copy the folder
+    docker cp "${container_name}:/home/openwrt/openwrt/artifacts/" "$build_dir"
+    mv "$build_dir/artifacts/"* "$build_dir"
```

Comment:
Maybe on `docker run` mount `$build_dir/artifacts` into `/home/openwrt/openwrt/artifacts` inside container to avoid operation of copying artifacts from container to host?